internal/function/language.go

```diff
@@ -1,12 +1,27 @@
 package function
 
 import (
+	"encoding/binary"
 	"fmt"
+	"hash/crc32"
 
+	lru "github.com/hashicorp/golang-lru"
 	enry "gopkg.in/src-d/enry.v1"
 	"gopkg.in/src-d/go-mysql-server.v0/sql"
 )
 
+const defaultLanguageCacheSize = 10000
```

Comment:
Right now, we are trying to let apps adjust cache sizes (and pass them down to packages like go-git), so maybe in this case we can also think about how easily we can modify this cache in the future (at least size).

Reply:
Yep, this should be configurable, right @ajnavarro? Environment variable or CLI flag that passes down to here? WDYT?

Reply:
yep, it should be configurable. it should be a env var o cli flag, yep.